### PR TITLE
[Webportal] Support Dshuttle storage

### PR DESCRIPTION
**Pull request by yiyione**
1. User profile page:
storage detail
2. Submit job page:
submit job
3. Storage detail:
user profile

### CODE REVIEW

File: src/webportal/src/app/user/fabric/user-profile/storage-list.jsx [hunk 1]
```diff
@@ -43,6 +43,15 @@ function getStorageServerUri(storageDetail) {
           {data.containerName}; <b>{'Path: '}</b>
         </>
       );
+    case 'dshuttle':
+      return (
+        <>
+          {'AzureBlob '}
```
Comment (Binyang2014) [inline src/webportal/src/app/user/fabric/user-profile/storage-list.jsx:49]: Can we show AzureBlob only when ufsType is `wasb`? For other type we can show the raw `ufsUri`

File: src/webportal/src/app/job-submission/utils/utils.js [hunk 2]
```diff
@@ -218,3 +218,11 @@ export function getStoragePlugin(extras) {
   }
   return plugins.find(plugin => plugin.plugin === 'teamwise_storage');
 }
+
+export function getDeshuttleStorageDetails(dshuttleMountPoint) {
```
Comment (Binyang2014) [inline src/webportal/src/app/job-submission/utils/utils.js:222]: The method only valid when ufs type is Azure. Can we change the return to {ufsType, accountName, containerName} when ufs type is Azure. And return {ufsType, ufsUri} when UFS type is others. And show difference in web UI

Comment (yiyione) [reply src/webportal/src/app/job-submission/utils/utils.js:222]: Done.

File: src/webportal/src/app/user/fabric/user-profile/storage-list.jsx (hunk 1 above)
Comment (yiyione) [reply src/webportal/src/app/user/fabric/user-profile/storage-list.jsx:49]: Done.

File: src/webportal/src/app/job-submission/components/data/team-detail.jsx [hunk 3]
```diff
@@ -353,4 +419,10 @@ export const SERVER_PATH = {
       {storage.data.path || '/'}
     </div>
   ),
+  dshuttle: storage => (
+    <div className={FontClassNames.semibold}>
+      <b>{`${storage.data.accountName}.blob.core.windows.net/${storage.data.containerName}`}</b>
+      {storage.data.path || '/'}
+    </div>
```
Comment (Binyang2014) [inline src/webportal/src/app/job-submission/components/data/team-detail.jsx:426]: Only return this when `storage.data.ufsType` is `wasb`. Else return `storage.data.ufsUri`?

File: src/webportal/src/app/job-submission/components/data/team-detail.jsx [hunk 4]
```diff
@@ -34,45 +34,82 @@ export default function TeamDetail({ isOpen = false, config, hide }) {
...
+      result.push({
+        key: 'ufsType',
+        name: 'UFS Type',
+        headerClassName: FontClassNames.semibold,
+        minWidth: 80,
+        onRender: item => {
+          if (item === undefined) {
+            return <div className={FontClassNames.small}>{'Invalid Type'}</div>;
+          } else {
+            return 'azureBlob';
```
Comment (Binyang2014) [inline src/webportal/src/app/job-submission/components/data/team-detail.jsx:76]: Return `azureBlob` when `storage.data.ufsType==wasb`. Else return `storage.data.ufsType` directly?

File: src/webportal/src/app/job-submission/components/data/team-detail.jsx (hunk 3 above)
Comment (yiyione) [reply src/webportal/src/app/job-submission/components/data/team-detail.jsx]: Done.

File: src/webportal/src/app/job-submission/components/data/team-detail.jsx (hunk 4 above)
Comment (yiyione) [reply src/webportal/src/app/job-submission/components/data/team-detail.jsx:76]: Done.

File: src/webportal/src/app/job-submission/models/data/mount-directories.js [hunk 5]
```diff
@@ -329,8 +329,8 @@ export class MountDirectories {
             'azurefile://' + data.accountName + '/' + data.shareName;
           break;
         case 'azureBlob':
-          returnValue =
-            'azureblob://' + data.accountName + '/' + data.containerName;
+        case 'dshuttle':
+          returnValue = `Azure Blob (accountName: ${data.accountName} containerName: ${data.containerName})`;
```
Comment (Binyang2014) [inline src/webportal/src/app/job-submission/models/data/mount-directories.js:333]: Here also need to change according to `ufsType`. Please change this?

**Review by Binyang2014 — APPROVED** [review]

File: src/webportal/src/app/job-submission/models/data/mount-directories.js (hunk 5 above)
Comment (yiyione) [reply src/webportal/src/app/job-submission/models/data/mount-directories.js]: Done.

**Pull request merged**